_readings/sister-outsider.md

```diff
@@ -0,0 +1,9 @@
+---
+identifier: sister-outsider
+title: "Sister Outsider"
+authors: "Audre Lorde"
+categories: race
+goodreads_url: https://www.goodreads.com/book/show/32951.Sister_Outsider
+amazon_url: https://www.amazon.com/Sister-Outsider-Speeches-Crossing-Feminist-ebook/dp/B006L7RCEI/
```

Comment:
Got it, thanks! I just created #43.

And that's a helpful description of when to use hyphens. I'll follow that going foward.r